cpp/src/slice2cpp/Gen.cpp

```diff
@@ -273,6 +273,9 @@ namespace
         return r;
     }
 
+    /// Returns a doxygen formatted link to the provided Slice identifier.
+    string cppLinkFormatter(string identifier) { return "{@link " + fixKwd(identifier) + "}"; }
+
```

Comment:
Apparently we weren't escaping names in C++ links...
Linking to an identifier that is a C++ keyword is a very niche case, but still good to fix.

Reply:
It was one of the bugs I was aware of when writing this PR.
But I went ahead and added a fix to this, so now it should be properly handled.